frontend/src/pages/destructionlist/hooks/useDataGridProps.tsx

```diff
@@ -48,6 +52,7 @@ export function useDataGridProps(
   paginatedResults: PaginatedDestructionListItems | PaginatedZaken,
   selectedResults: (Zaak | { url: string })[],
   actions?: DataGridAction[],
+  zaaktypeChoicesQueryParams?: ZaaktypeChoicesQueryParam,
```

Comment:
Suggestion:

Replace `zaaktypeChoicesQueryParams` with two separate parameters:

```
destructionList?: DestructionList;
review?: Review
```

And have this hook figure out what to use.

Motivation:

This may keep things cleaner in the future duplication when some other functionality needs to access values from the destruction list and / or review.